openmrs-19/src/test/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceTest.java

```diff
@@ -209,4 +218,25 @@ private Concept createTestConcept() {
 
         return concept;
     }
+
+    private List<Observation> convertObservationMapToList(Map<String, String> observations, DateTime obsDatetime) {
```

Comment:
this method is redundant, because it is test and you know how observationsMap looks like.